bootstrap/src/main/resources/application.conf

```diff
@@ -2,11 +2,11 @@ authentication {
 
   db {
     driver = "org.postgresql.Driver"
-    url = "jdbc:postgresql://localhost:5432/random_coffee?currentSchema=authentication"
+    url = "jdbc:postgresql://"${RANDOM_COFFEE_DB_SERVER_NAME}":5432/coffee_db?currentSchema=authentication"
     user = "postgres"
-    user = ${?AUTH_DB_USER}
+    user = "postgres"
+    password = "postgres"
     password = "postgres"
```

Comment:
I think you should use parameters from an additional env or conf file with {SOME_PARAMETER}. Because if you create a dev or prod pipeline, you can use ansible or something else to create that.